defaults.yml

```diff
@@ -228,9 +228,8 @@ phpcs:
   # Space-separated list of directories to review.
   directories: "${drupal.root}/modules/custom ${drupal.root}/themes/custom ${drupal.root}/profiles/custom"
 
-  # Comma-separated list of patterns for files and directories to exclude from the
```

Comment:
Existing build.xml files use this property -- so we should keep it, but add a comment saying it will be deprecated in 3.0 (which... we don't have a roadmap really so who knows when that would be...)